### ISSUE TITLE
Combobox should have its own scrollable section for options to allow smooth keyboard navigation

### ISSUE THREAD

**Issue by ayelethol** [mondaycom/monday-ui-react-core#155]
Currently when we use the combobox component we need to wrap it in a scrollable div.
This results in the keyboard navigation of the combobox not controlling the scrollable part and we lose view of the focused items when they are under the scroll line.
The combobox component should have its own prop of maxHeight for the options section and control the scrollable when using keyboard navigation to make sure the focused item is always in view.
cc @adirhaz1z
